frame/system/src/lib.rs

```diff
@@ -1753,7 +1764,7 @@ impl<T: Config> Lookup for ChainContext<T> {
 
 /// Prelude to be used alongside pallet macro, for ease of use.
 pub mod pallet_prelude {
-	pub use crate::{ensure_none, ensure_root, ensure_signed, ensure_signed_or_root};
```

Comment:
You still have to export it, otherwise this is a braking change and Polkadot will not find it anymore.